mne/viz/backends/_qt.py

```diff
@@ -516,6 +510,7 @@ def _window_initialize(self):
         super()._window_initialize()
         self._interactor = self.figure.plotter.interactor
         self._window = self.figure.plotter.app_window
+        self._window_load_icons()
         self._window_set_theme()
```

Comment:
Does it have to come after the set_theme so that the correct theme is loaded?

Reply:
`_window_set_theme()` updates the icons as well so the order should not matter in here